docs/advisories/etcd-manager-certificate-expiration.md

```diff
@@ -0,0 +1,69 @@
+# etcd-manager Certificate Expiration
+
+etcd-manager configures certificates for TLS communication between kube-apiserver and etcd, as well as between etcd members.
+These certificates are signed by the cluster CA and are valid for a duration of 1 year.
+
+Affected versions of etcd-manager currently do NOT automatically rotate these certificates before expiration.
+If these certificates are not rotated prior to their expiration, Kubernetes apiserver will become inaccessible and your control-plane will experience downtime.
+
+## How do I know if I'm affected?
+
+Clusters are affected by this issue if they're using a version of etcd-manager < `3.0.20200428`.
+The etcd-manager version is set automatically based on the Kops version.
+These Kops versions are affected:
+
+* Kops 1.10.0-alpha.1 through 1.16.1
```

Comment:
Safe versions are going to be 1.15.3, 1.16.2, 1.17.0-beta.2, 1.18.0-alpha.3

Releases of 1.15.3 and 1.16.2 are staged but not published (please take a look at the message).  I figured we should probably do them all at once...

Reply:
ok, i updated the list of affected versions and the list of fixed versions in the solutions section below.

Reply:
I updated the release drafts to link to kops.sigs.k8s.io, and confirmed this page looks good [on netlify](https://deploy-preview-9030--kubernetes-kops.netlify.app/advisories/etcd-manager-certificate-expiration/).